src/reactor/reactor.spec.ts

```diff
@@ -541,6 +541,175 @@ describe('reactor/reactor with cycles', () => {
     });
 });
 
+describe('reactor/reactor error handling', () => {
+    let a$: Atom<string>;
+    let d$: Derivable<string>;
+
+    beforeEach('setup atom and derivable', () => {
+        a$ = atom('no error');
+        d$ = a$.derive(v => {
+            if (v === 'error in derivation') {
+                throw new Error(v);
+            }
+            return v;
+        });
+    });
+
+    context('with no error handler provided', () => {
+        context('when an error occurs in any derivation', () => {
+            beforeEach('start the reactor', () => {
+                react(d$);
+                shouldHaveReactedOnce('no error');
+            });
+
+            it('should throw on Atom#set', () => {
+                expect(() => a$.set('error in derivation')).to.throw('error in derivation');
+            });
+
+            it('should stop the reactor', () => {
+                a$.set('whatever');
+                shouldHaveReactedOnce('whatever');
+
+                try { a$.set('error in derivation'); } catch (e) { /**/ }
+                shouldNotHaveReacted();
+
+                a$.set('no error at all!');
+                shouldNotHaveReacted();
+            });
+        });
+
+        context('when an error occurs in any reactor', () => {
+            let latestValue: string;
+            beforeEach('start an unstable reactor', () => {
+                d$.react(v => {
+                    latestValue = v;
+                    if (v === 'error in reactor') {
+                        throw new Error(v);
+                    }
+                });
+            });
+
+            it('should throw on Atom#set', () => {
+                expect(() => a$.set('error in reactor')).to.throw('error in reactor');
+            });
+
+            it('should stop the reactor', () => {
+                a$.set('whatever');
+                expect(latestValue).to.equal('whatever');
+
+                try { a$.set('error in reactor'); } catch (e) {/**/ }
+                expect(latestValue).to.equal('error in reactor');
+
+                a$.set('back to normal?');
+                expect(latestValue).to.equal('error in reactor');
+            });
+
+            it('will break other reactors, and therefore your application', () => {
```

Comment:
'break' can be confusing in this context. Since it can mean that reactor's won't react anymore.
Maybe use something like: 'cause other reactors to fail'.